src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -115,15 +89,27 @@ class WorkspaceReimburseView extends React.Component {
         this.fetchData();
     }
 
-    getUnitRateValue(customUnitRate) {
+    getCurrentRatePerUnitLabel() {
+        const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        const customUnitRate = _.find(lodashGet(distanceCustomUnit, 'rates', {}), rate => rate.name === 'Default Rate');
+        const currentUnit = this.getUnitLabel(lodashGet(distanceCustomUnit, 'attributes.unit', 'mi'));
+        const currentRate = this.getRateLabel(customUnitRate);
+        const perWord = this.props.translate('workspace.reimburse.per');
+        return `${currentRate} ${perWord} ${currentUnit}`.trim();
+    }
+
+    getRateLabel(customUnitRate) {
         return this.getRateDisplayValue(lodashGet(customUnitRate, 'rate', 0) / CONST.POLICY.CUSTOM_UNIT_RATE_BASE_OFFSET);
     }
 
-    getUnitItems() {
-        return [
-            {label: this.props.translate('workspace.reimburse.kilometers'), value: 'km'},
-            {label: this.props.translate('workspace.reimburse.miles'), value: 'mi'},
-        ];
+    getUnitLabel(value) {
```

Comment:
I think we can just add the following to common localization:
```
'mi': 'mile',
'km': 'kilometer'
```
and then getUnitLabel would do
```return this.props.translate(`common.${unit}`)```

Reply:
I changed the key translation kilometer to km and mile to mi.